tests/unit/general/test_umpire_mem.cpp

```diff
@@ -162,7 +174,8 @@ static void test_umpire_device_memory()
    // Just as an example, temp memory on the stack is automatically cleaned up
    {
       printf("Allocate %u more bytes in temporary memory: ", num_bytes);
-      Vector dev_temp(num_elems, mc::DEVICE_TEMP);
+      Vector dev_temp(num_elems, MemoryType::DEVICE_UMPIRE_2);
```

Comment:
Here, an application could make an alias for ``MemoryType::DEVICE_UMPIRE_2``

Reply:
That's definitely what we'll do!